scripts/generate_dump

```diff
@@ -47,7 +49,15 @@ handle_signal()
 }
 trap 'handle_signal' SIGINT
 
+handle_error() {
+  if [ "$1" != "0" ]; then
+    echo "ERR: RC:-$1 observed on line $2" >&2
+    RETURN_CODE=1
+  fi
```

Comment:
>fi [](http://example.com/codeflow?start=2&length=2)

In your PR description, this function `return $1`, which is different with the code.